### ISSUE TITLE
Feature request: Ability to send arbitrary commands from dashboard

### ISSUE THREAD

**Issue by sgryphon** [home-assistant-libs/python-matter-server#816]
In the UI, as well as exploring nodes and viewing (reading) attribute values, it would be good if there was a way to send arbitrary commands directly (specifically device commands). 

This could be hidden behind an Advanced mode (like Home Assistant administration) if there was worry that it could mess things up.

Example:

device command args:
  node_id = 12
  endpoint_id = 1
  cluster_id = 0x0006 (On/Off cluster)
  command_name = "On" (0x01)
  payload = null 

Device commands could be raw IDs, or friendly names from the specs (as above). The commands accepted by each cluster are available in the data:

e.g. For a Nanoleaf light the AcceptedCommandList on cluster 0x6 includes Off, On (0x1), Toggle, OffWithEffect (hex 0x40 = 64), etc

![Screenshot 2024-07-19 145802](https://github.com/user-attachments/assets/f90f4a20-500b-48d0-a43f-f982a5a92501)

Some commands, e.g. the On/Off cluster commands for On and Off are fairly simple and don't have any data fields, so should be relatively straight forward.

A command like OffWithEffect (0x40) might be more complex to support various arguments.

A completely custom command (not in the Matter Spec, i.e. for a custom device) would be even more difficult.

**Comment by marcelveldt** [comment]
Yes, we have in mind to add this but its low prio for the moment. 
We have so many other stuff to concentrate on first.

Idea is that you can write attribute values (for attributes that support write) and that we list the commands for the cluster and be able to issue them.
